src/bp/studio/studio-router.ts

```diff
@@ -104,6 +104,13 @@ export class StudioRouter extends CustomRouter {
   async setupRoutes(app: express.Express) {
     this.botpressConfig = await this.configProvider.getBotpressConfig()
 
+    this.actionsRouter.setupRoutes()
+    this.flowsRouter.setupRoutes()
+    this.logsRouter.setupRoutes()
+    await this.mediaRouter.setupRoutes(this.botpressConfig)
+    this.notificationsRouter.setupRoutes()
+    this.topicsRouter.setupRoutes()
+
```

Comment:
is that new ? how is this related to your PR except ln#110

Reply:
This is the workaround to make sure we don't have any floating promises in one of our routers constructor. This change also allows passing the same botpressConfig object instead of having each router fetching it.

Reply:
Got it.

as you know bp config is cached so you won't notice any performance gain here but that's ok.